packages/react/src/modules/guide/components/Modal/Modal.tsx

```diff
@@ -108,6 +111,21 @@ const Body: React.FC<{ body: string } & React.ComponentPropsWithRef<"div">> = ({
 };
 Body.displayName = "ModalView.Body";
 
+const Img: React.FC<
```

Comment:
Worth extracting any of this into shared image components that take a type? Seems fine duplicating them at this point.

Reply:
Yeah I think keeping them separate feels fine to me at this time, not a ton going on with this component to make me feel like we need to abstract things away.